src/js/background/assignManager.js

```diff
@@ -490,6 +490,9 @@ const assignManager = {
         currentCookieStoreId = backgroundLogic.cookieStoreId(currentUserContextId);
         confirmUrl += `&currentCookieStoreId=${currentCookieStoreId}`;
       }
+      else {
+        currentCookieStoreId = "firefox-default";
```

Comment:
See my other comments however if we need this I would prefer all non test usage has a global.